README.md

```diff
@@ -28,6 +32,15 @@ type: docs
 * inclusion of `type: docs` is important for the theme to properly index the site contents
 * the `linkTitle` attribute allows you to simplify the name as it appears in the left-side nav bar - ideally it should be short and clear - whereas the title can handle longform names for pages/documents.
 
+## Adding release-specific docs
+
+Steps to add a release-specific version of the docs site:
+
+1. Create a release branch from main, named after the released major and minor version, like [release-v0.8](https://github.com/openservicemesh/osm-docs/tree/release-v0.8).
```

Comment:
In addition to creating the branch `release-vX.Y`, the branch should also be pushed `upstream` to this repo in order for it to be picked up by Netlify. 

By default, Netlify will deploy the branch to a url like https://release-v0-8--osm-docs.netlify.app/ which can be used to preview and test that it builds correctly. Though that testing could be carried out under the third bullet point

Reply:
Thanks. I've edited and hopefully clarified; please check again for accuracy? Thanks.